docs/_templates/search.html

```diff
@@ -15,173 +15,12 @@
     </div>
 
     <nav id="pagination" class="pagination" aria-label="pagination" hidden="true" >
-        <button class="pagination-previous" id="pagination-previous">
-            ← Previous
-        </button>
+        <button class="pagination-previous" id="pagination-previous" data-pagination-previous></button>
```

Comment:
Hi @lb- , The reason I was forced to do create a Next and Previous button dynamically is because I replaced onclick with AddEventListerners and the feature of this is that it gets triggered for all the times it was mentioned in the DOM tree, whereas onclick gets triggered only for the last one. Hence, I created a Next/Previous button and dynamically replaced it on every page, so that the Listener event gets triggered only once.
Let me know if I was able to explain this, and also if you have any suggestions to circumvent this problem.

Reply:
Hmm. Good explanation - I'll have a better look tomorrow and see if we can find a simpler way. 

Thanks for the response and your persistence on getting this through.

Reply:
Thank you so much. Meanwhile, I'll also keep at it to see if I find a solution.

Reply:
@lb- , managed to solve this roadblock using removeEventListener functions. Also, I had to take some operations in new functions, to follow the correct format of using these remove event functions. Please do have a look at the code and let me know your thoughts.
Thanks!

Reply:
@lb- Can't Find Search.js File Mentioned In This PR